test/testclangimport.cpp

```diff
@@ -878,7 +878,7 @@ class TestClangImport: public TestFixture {
                       "\n"
                       "ns::abc x@1 = c ;", parse(clang));
 
-        GET_SYMBOL_DB(clang);
+        GET_SYMBOL_DB(clang)
```

Comment:
This is not "proper" syntax. I would prefer that the semicolon was required. Maybe the macro can be rewritten instead.

Reply:
Yeah. I didn't like that either, but the whole code in that macro has to be in the scope. I could just add some additional code which does nothing at the end.

Reply:
sure .. tweak the macro somehow...

Reply:
done